recipe/meta.yaml

```diff
@@ -12,14 +12,26 @@ source:
   patches:
     - vc14.patch
 
+# While Metis is a pure C library, it is built with cmake, which assumes C++
+# by defaults and attempts to locate the C++ compiler. Hence the recipe has a
+# build dependency on {{ compiler('cxx') }}. The `ignore_run_exports` entry is
+# added so that the package does not depend on the C++ runtime due to the
+# dependency to the C++ compiler.
+#
+# Another fix to this would be to have metis' cmake specify `project(METIS C)`
+# instead of `project(METIS)`.
+
 build:
   number: 1003
   binary_relocation: true
+  ignore_run_exports:        # Cf comment above
+    - {{ compiler('cxx') }}
```

Comment:
I don't think this works.

This needs to be,
```
- libstdcxx-ng   # [linux]
- libcxx   # [osx]
```

Reply:
What is the issue with `compiler('cxx')`?

Reply:
Do `conda render recipe` and see the output. `libstdcxx-ng` is still a runtime dependency.

Reply:
Was about to raise an issue for this and saw you beat me to it. 😄

xref: https://github.com/conda/conda-build/issues/3252